src/arkode/arkode_impl.h

```diff
@@ -45,6 +45,7 @@ extern "C" {
   SHORTCUTS
   ===============================================================*/
 
+#define ARK_SUNCTX ark_mem->sunctx
 #define ARK_PROFILER ark_mem->sunctx->profiler
 #define ARK_LOGGER ark_mem->sunctx->logger
```

Comment:
Do we need these shortcut macros to `ark_mem->sunctx->profiler` and `ark_mem->sunctx->logger` as well?